quipucords/api/common/util.py

```diff
@@ -65,6 +73,8 @@ def convert_to_boolean(value):
     :param value: The value to convert
     :return The value as a bool
     """
+    if isinstance(value, bool):
```

Comment:
I don't think we need to do this check since it is inside of function is_boolean which we immediately call

Reply:
we do or it will explode because bool has not method `value.lower() `